### PR TITLE
byoc/playground config validation

### PR DESCRIPTION
**Pull request by nullfunc**
Ported from mvp version of validation of project configs

### CODE REVIEW

File: src/pkg/cli/compose/validation.go [hunk 1]
```diff
@@ -29,6 +39,11 @@ func ValidateProject(project *composeTypes.Project) error {
 	sort.Slice(services, func(i, j int) bool {
 		return services[i].Name < services[j].Name
 	})
+
+	if err := ValidateProjectConfig(context.Background(), project, listConfigNamesFunc); err != nil {
```
Comment (lionello) [inline src/pkg/cli/compose/validation.go:43]: `context.TODO()` because we really should be using the caller's context here. `context.Background()` is fine in some corner cases, and in test code.
